app/assets/javascripts/javascripts/mailchimp_newsletter.js

```diff
@@ -0,0 +1,15 @@
+var mailChimpNewsletter = function(){
+  var init = function(formSelector, responseSelector){
+    $(formSelector).ajaxChimp({
+      url: "http://whatweate.us9.list-manage.com/subscribe/post?u=3ae9a01a53d464033afcc3009&id=eb7d06a555",
+      callback: function(data){
+        ga("send", "event", "newsletter", "submit", "fill in newsletter");
+        $(responseSelector).text(data.msg);
+      }
+    });
+  };
+
+  return {
+    init: init
+  }
+}();
```

Comment:
Wrap an immediate function invocation in parens to assist the reader in understanding that the expression is the result of a function, and not the function itself.